package.json

```diff
@@ -175,6 +175,7 @@
     "scroll-frame": "1.0.0",
     "should": "^11.1.1",
     "sinon": "^1.17.1",
+    "source-map-support": "^0.4.14",
```

Comment:
Added this because Circle was failing and returning stack traces that pointed to the wrong line numbers. Hopefully this is helpful. I didn't get a chance to find out because the tests past as soon as I added it in. :man_shrugging: